docs/v1.1.0/schema.md

```diff
@@ -0,0 +1,216 @@
+| Annotation object             | ???                      | N/A                               |                             | STRING     |  --  |   --    | MUST                                                                  |                                                                   | Name of the object being annotated (e.g. cytosolic ribosome, nuclear pore complex, actin filament, membrane)                                                                                                                                                                                                                       |
+| Annotation object ID          | ???                      | N/A                               |                             | STRING     |  --  |   --    | MUST                                                                  | Gene Ontology                                                     | Gene Ontology Cellular Component identifier for the annotation object, e.g. http://purl.obolibrary.org/obo/GO_0022626 for ‘cytosolic ribosome’.                                                                                                                                                                                    |
+| Annotation object description | ???                      | N/A                               |                             | TEXT       |  --  |   --    | OPTIONAL                                                              |                                                                   | A textual description of the annotation object, can be a longer description to include additional information not covered by the Annotation object name and state.                                                                                                                                                                 |
+| Annotation object state       | ???                      | N/A                               |                             | STRING     |  --  |   --    | OPTIONAL                                                              |                                                                   | Molecule state annotated (e.g. open, closed)                                                                                                                                                                                                                                                                                       |
```

Comment:
In the google schema these are top level keys, but in the files they are nested.